build.sbt

```diff
@@ -5,7 +5,7 @@ name := "soy"
 organization := "com.kinja"
 
 // We use Semantic Versioning. See: http://semver.org/
-version := "0.4.4" + {if (System.getProperty("JENKINS_BUILD") == null) "-SNAPSHOT" else ""}
+version := "0.5" + {if (System.getProperty("JENKINS_BUILD") == null) "-SNAPSHOT" else ""}
```

Comment:
The public API is stable and has been for a while. We shouldn't keep this in beta forever just so we can avoid proper versioning.